lib/northbound_grpc.cpp

```diff
@@ -92,33 +94,41 @@ class Candidates
 		return c;
 	}
 
-	void delete_candidate(struct candidate *c)
+	bool contains(uint64_t candidate_id)
+	{
+		return _cdb.count(candidate_id) > 0;
+	}
+
+	void delete_candidate(uint64_t candidate_id)
```

Comment:
This leads to a an issue in class destructor.
```	
~Candidates(void)
{
// Delete candidates.
for (auto it = _cdb.begin(); it != _cdb.end(); it++)
	delete_candidate(&it->second);
}
```

 ```lib/northbound_grpc.cpp: In destructor ‘Candidates::~Candidates()’:
lib/northbound_grpc.cpp:81:21: error: invalid conversion from ‘candidate*’ to ‘uint64_t’ {aka ‘long unsigned int’} [-fpermissive]
   81 |    delete_candidate(&it->second);
      |                     ^~~~~~~~~~~
      |                     |
      |                     candidate*
lib/northbound_grpc.cpp:102:33: note:   initializing argument 1 of ‘void Candidates::delete_candidate(uint64_t)’
  102 |  void delete_candidate(uint64_t candidate_id)
      |                        ~~~~~~~~~^~~~~~~~~~~~
make[1]: *** [Makefile:14393: lib/grpc_la-northbound_grpc.lo] Error 1```

Reply:
is fixed now, thanks.